instrumentation/opentelemetry-instrumentation-falcon/src/opentelemetry/instrumentation/falcon/__init__.py

```diff
@@ -243,6 +251,11 @@ class _InstrumentedFalconAPI(getattr(falcon, _instrument_app)):
     def __init__(self, *args, **kwargs):
         otel_opts = kwargs.pop("_otel_opts", {})
 
+        _OpenTelemetrySemanticConventionStability._initialize()
```

Comment:
I think we are missing some important points here:

# Checklist
- [X] Add semconv status as `migration` for urllib3 in instrumentations README
- [X] Populate `{method}` as `HTTP` when nonstandard method (sanitize_method)
- [X]  set request HTTP method attribute as `_OTHER` when nonstandard http method and for new semconv also set `http.request.method_original`  with the original nonstandard method.
- [X] Set schema_url based on the opt-in mode
- [X] Create histograms based on the opt-in mode (different metric names so no dup attributes)
- [X] Set metrics attributes based on the opt-in mode
- [X] Set span attributes based on the opt-in mode

Reply:
Thank you for the great feedback, I'll get started on this.

Reply:
LGTM. Left a small comment